CONTRIBUTING.md

```diff
@@ -42,3 +42,30 @@ _This example is for launching Brave on MacOS_
 ```bash
 npm run-script start:chromium -- --chromium-binary /Applications/Brave\ Browser.app/Contents/MacOS/Brave\ Browser
 ```
+
+### Safari
+
+Safari [only supports web extensions](https://developer.apple.com/documentation/safariservices/safari_web_extensions) bundled inside a native Mac app. If you are *not* building for development, Safari [only loads web extensions if their containing apps are distributed via the Mac App Store](https://developer.apple.com/documentation/safariservices/safari_web_extensions/distributing_your_safari_web_extension), or [if you select “Allow unsigned extensionsʺ from the Debug menu on each launch](https://developer.apple.com/documentation/safariservices/safari_web_extensions/running_your_safari_web_extension) (requires authentication).
+
+To build for development:
+
+1. Build the extension for manifest version 2:
+
+    ```bash
+    npm run build:mv2
```

Comment:
I think this should be `npm run build:mv2-safari`.

Reply:
Oops, fixed.